site/_data/navigation.yml

```diff
@@ -18,8 +18,14 @@ topnav:
 - title: Get Started
   url: /get-started
 
-- title: API Docs
-  url: https://github.com/apache/incubator-livy#rest-api
+- title: Documentation
+  subcategories:
+    - title: REST API
+      url: https://github.com/apache/incubator-livy/blob/master/docs/rest-api.md#rest-api
+    - title: Programmatic API
+      url: https://github.com/apache/incubator-livy/blob/master/docs/programmatic-api.md#using-the-programmatic-api
```

Comment:
yeah that's the idea, this pR is to just add the examples and update the links to the new doc file locations. Then in the followup PR I'll setup jekyll to build the docs and use those built docs instead of the links to the repo.